hw_3/hw_3/ViewController.swift

```diff
@@ -0,0 +1,177 @@
+        let numberSum = firstnumber + secondNumber
+        print("\(firstnumber) + \(secondNumber) = \(numberSum) ")
+    }
+    
+    func subtraction(_ firstnumber: Int,_  secondNumber: Int) {
+        let numberSum = firstnumber - secondNumber
+        print("\(firstnumber) - \(secondNumber) = \(numberSum) ")
+    }
+    
+    func subtraction(_ firstnumber: Int,_  secondNumber: Double) {
+        let numberSum = Double(firstnumber) - secondNumber
+        print("\(firstnumber) - \(secondNumber) = \(numberSum) ")
+    }
+    
+    func subtraction(_ firstnumber: Double,_  secondNumber: Int) {
+        let numberSum = firstnumber - Double(secondNumber)
+        print("\(firstnumber) - \(secondNumber) = \(numberSum) ")
+    }
+    
+    func subtraction(_ firstnumber: Double,_  secondNumber: Double) {
+        let numberSum = firstnumber - secondNumber
+        print("\(firstnumber) - \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstnumber: Int,_  secondNumber: Int) {
+        let numberSum = firstnumber * secondNumber
+        print("\(firstnumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstnumber: Int,_  secondNumber: Double) {
+        let numberSum = Double(firstnumber) * secondNumber
+        print("\(firstnumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstnumber: Double,_  secondNumber: Int) {
+        let numberSum = firstnumber * Double(secondNumber)
+        print("\(firstnumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstnumber: Double,_  secondNumber: Double) {
+        let numberSum = firstnumber * secondNumber
+        print("\(firstnumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func devision(_ firstnumber: Int,_  secondNumber: Int) {
+        let numberSum = Double(firstnumber) / Double(secondNumber)
+        print("\(firstnumber) / \(secondNumber) = \(numberSum) ")
+    }
+    
+    func devision(_ firstnumber: Int,_  secondNumber: Double) {
+        let numberSum = Double(firstnumber) / secondNumber
+        print("\(firstnumber) / \(secondNumber) = \(numberSum) ")
+    }
+    
+    func devision(_ firstnumber: Double,_  secondNumber: Int) {
+        let numberSum = firstnumber / Double(secondNumber)
+        print("\(firstnumber) / \(secondNumber) = \(numberSum) ")
+    }
+    
+    func devision(_ firstnumber: Double,_  secondNumber: Double) {
+        let numberSum = firstnumber / secondNumber
+        print("\(firstnumber) / \(secondNumber) = \(numberSum) ")
+    }
+
+    func stringComparison(firstString: String, secondString: String) {
+        if firstString > secondString {
+            print("\(firstString) > \(secondString)")
+        }
+        else if firstString < secondString {
+            print("\(firstString) < \(secondString)")
+        }
+        else {
+            print("\(firstString) = \(secondString)")
+        }
+    }
+    
+    func sumFour(number: Int) {
```

Comment:
I think we can improve `sumFour` name here. We sum digits of a four-digit number here. So we can name it like `sumFourDigitNumberDigits`. Yes, it's long and repeats `digit` in the name but you can't replace this word with another and it became clearer what this function does